sbatch_scripts/gen_sbatch.py

```diff
@@ -31,41 +31,44 @@
 python -m uv run {command}
 """
 
-
 def create_command(command_name: str, **params: dict):
     command = f"{command_name} "
     command += " ".join([f"--{key} {value}" for key, value in params.items()])
     return command
 
 
 def create_sbatch_script(
+    account="default",
+    partition="batch",
+    num_gpus=0,
```

Comment:
Let's make this have a default value of 1, the typical use case for GPU workflows.
```suggestion
    num_gpus=1,
```

Reply:
@cpaniaguam I set the default value of this to 0 instead of 1 because the `num_gpus` argument is a part of the SBATCH metadata regardless of the partition being used (batch or gpu), so it'll give you an error if you specify needing GPUs on the batch partition. Let me know if there's a better way to do this

Reply:
This is probably the easiest way. Alex sort of mentioned it'd be nice to have one fewer argument to specify. I thought it would default to 0 gpu is one requested the batch partition but Slurm doesn't seem to like that and making it more flexible will require changing the template and adding extra logic. I say we keep it as is :)